documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -437,3 +355,82 @@ the operator REST API,
 setting operator pod labels,
 setting operator pod annotations,
 and Istio.
+
+### Special use cases
+
+If applicable, please review the following special use cases.
+
+#### How to download the Helm chart if Internet access is not available
+
+At a high-level, you use `helm pull` to download a released version of the Helm chart and move it to the machine with no Internet access,
+so that then you can run `helm install` to install the operator.  
+
+The steps are:
+1. On a machine with Internet access, to download the chart to the current directory, run `$ helm pull weblogic-operator --repo https://oracle.github.io/weblogic-kubernetes-operator/charts --destination .`
+2. Move the resulting `weblogic-operator-<version>.tgz` file to the machine without Internet access on which you want to install the WebLogic Kubernetes Operator.
+3. Run `$ tar zxf weblogic-operator-<version>.tgz`. This puts the Helm chart files in a local directory `./weblogic-operator`.
+4. To create the namespace where the operator will be installed, run `$ kubectl create namespace weblogic-operator`. **NOTE**: Be sure to follow all
```

Comment:
The NOTE should make it clear that more steps may be involved other than just setting up the NS and service-account.  Maybe something like "**NOTE**: Be sure to follow all of the previously detailed prerequisite steps, starting at Inspect the operator Helm chart and ending with Be aware of advanced operator configuration options." ?

Reply:
@tbarnes-us IMO, I think that stating, "Be sure to follow **all** of the previously detailed prerequisite steps..." includes them all, up to the Special use cases section. However, if you feel strongly that I need to be more specific about the steps, I'll add the information.